### DISCUSSION TITLE
New task status

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#32] [Shipped - Live on production]
Currently there are 3 task status for repair tasks: New, Pending, Done. I. would like to request additional status. New I see as problems that just have been reported and we have not investigated anything yet. Done obviously is when the job is finished. Pending i would like to see as tasks that are waiting on others (delivery of parts, getting outside service, etc).

I would like to request **to add a status** that indicates we are working on it (our in-house biomeds); so maybe something like **In-Work** or so. In developing contexts devices are often out of operation for a long time because sourcing parts or external service takes very long for various reasons. So many tasks will be pending, but only some of these pending tasks will be actually in our own hands. And it is helpful to quickly see where to focus our attention on.

**Comment by BioMedGPH** [comment]
It would be really great if this would be included in a next release as well!

**Comment by YvesR** [comment]
We consider this in the next release as we also thinking about an inventory (operation)-status

**Reply by BioMedGPH** [reply]
Any update on this In-work status for operations?

**Reply by YvesR** [reply]
this is live on production for incidents and issues.
